### PR TITLE
Add ARM64 support to pkg/sentry/loader

### PR DESCRIPTION
**Pull request by lubinszARM**
Signed-off-by: Bin Lu <bin.lu@arm.com>

### CODE REVIEW

**Comment by lubinsz** [comment]
Hi @prattmic 
Any comments?
Thanks.

File: pkg/sentry/loader/elf.go [hunk 1]
```diff
@@ -560,6 +566,13 @@ func loadInitialELF(ctx context.Context, m *mm.MemoryManager, fs *cpuid.FeatureS
 		return loadedELF{}, nil, err
 	}
 
+	// Check Image Compatibility.
+	ok := strings.EqualFold(arch.Host.String(), info.arch.String())
```
Comment (prattmic) [inline pkg/sentry/loader/elf.go]: Why isn't this just arch.Host == info.arch?

Comment (lubinsz) [reply pkg/sentry/loader/elf.go]: Ensure sufficient robustness. Do you want to change it?

Comment (lubinsz) [reply pkg/sentry/loader/elf.go]: Done.

**Pull request merged**